.eslintrc

```diff
@@ -12,6 +12,7 @@
   },
   "parser": "babel-eslint",
   "rules": {
+    "arrow-body-style": ["off", "always"],
```

Comment:
I think an update caused it to start yelling at me so I changed it (or maybe it was just a tonne of warnings).

I will do it if it doesn't cause failures though, sure thing.